pkgs/top-level/all-packages.nix

```diff
@@ -25441,10 +25441,11 @@ with pkgs;
 
   em = callPackage ../applications/editors/em { };
 
-  emacs = emacs27;
-  emacs-nox = emacs27-nox;
+  emacs = emacs28;
+  emacsNativeComp = emacs28NativeComp;
```

Comment:
Could you use the attribute `emacsGcc` instead as that was already established by https://github.com/nix-community/emacs-overlay.

Reply:
Done.

Reply:
Gah.. I changed my mind about this.
I think we should set it to `emacsNativeComp` instead as it's the upstream name.

Iirc the reason why it's called `emacsGcc` in the overlay is because the derivation predates the whole `nativeComp` naming.

I've changed it back and force-pushed. Sorry for the noise.